torchvision/csrc/io/decoder/video.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
```

Comment:
This code looks same as https://github.com/pytorch/vision/blob/main/torchvision/csrc/io/video/video.h.